cvat/apps/annotation/README.md

```diff
@@ -564,4 +566,60 @@ python create_pascal_tf_record.py --data_dir <path to VOCdevkit> --set train --y
       └── image2.png
   ```
 - supported shapes: Polygons
-- additional comments: the CVAT task should be created with the full label set that may be in the annotation files
\ No newline at end of file
+- additional comments: the CVAT task should be created with the full label set that may be in the annotation files
+
+### [MOT sequence](https://arxiv.org/pdf/1906.04567.pdf)
+#### Dumper
+- downloaded file: a zip archive of the following structure:
+  ```bash
+  taskname.zip/
+  ├── images/
+  |   └── img1.jpg
+  └── gt/
+      ├── labels.txt
+      └── gt.txt
+
+  # labels.txt
+  cat
+  dog
+  person
+  ...
+
+  # gt.txt
+  # frame_id, track_id, x, y, w, h, not ignored, class_id, visibility, <ignored>
+  1,1,1363,569,103,241,1,1,0.86014
+  ...
+
+  ```
+- supported annotations: Rectangle shapes and tracks
+- supported attributes: `visibility` (number), `ignored` (checkbox)
+
+#### Loader
+- uploaded file: a zip archive of the structure above or:
+  ```bash
+  taskname.zip/
+  ├── labels.txt # for non-official labels
```

Comment:
To be honest I don't understand  the comment. Could you please re-phrase it somehow? Can it contain official labels?

Reply:
It can, but it is not required, if labels are official.